openapi.yaml

```diff
@@ -16,176 +16,171 @@ host: api.oregonstate.edu
-      prev:
+        description: Application number
+        example: '3'
+      statusCode:
         type: string
-        format: url
-        description: The previous page of data
-      next:
+        description: Status code
+        example: 'D'
+      statusDate:
         type: string
-        format: url
-        description: The next page of data
-  PetResource:
-    properties:
-      id:
+        description: |
+          'The status date uses the date-time notation as defined by RFC 3339 in Pacific Time Zone'
+        example: '2019-01-01T00:00:00+08:00'
+      decisionCode:
         type: string
-        description: A unique ID of pet
-      type:
+        description: Application decision code
+        example: 'AT'
+      decisionDate:
         type: string
-        enum: [pet]
-      links:
-        $ref: '#/definitions/SelfLink'
-      attributes:
-        properties:
-          name:
-            type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
-            type: string
-            description: Name of owner
-            example: Harry Potter
-          species:
-            type: string
-            description: Species of pet
-            example: Owl
-  PetResult:
-    properties:
-      links:
-        $ref: '#/definitions/SelfLink'
-      data:
-        $ref: '#/definitions/PetResource'
-  PetsResult:
+        format: date-time
+        description: |
+          'The decision date uses the date-time notation as defined by RFC 3339 in Pacific Time Zone'
+        example: '2019-01-01T00:00:00+08:00'
+      levelCode:
+        type: string
+        description: Application level code
+        example: '02'
+      campusCode:
+        type: string
+        description: Campus code
+        example: 'DSC'
+      studentTypeCode:
+        type: string
+        description: Student type code
+        example: 'T'
+      admitCode:
+        type: string
+        description: Admit code
+        example: 'RA'
+      initialCompleteDate:
+        type: string
+        format: date-time
+        description: |
+          'The initial complete date uses the date-time notation as defined by RFC 3339 in Pacific Time Zone'
+        example: '2019-01-01T00:00:00+08:00'
+      justCompletedInd:
+        type: boolean
+        description: The indicator shows if the application is just completed
+        example: true
+      uacPendingInd:
+        type: boolean
+        description: The indicator shows if the application is strill pending in University Admission Committee
+        example: true
+  ApplicationResult:
     properties:
       links:
         allOf:
           - $ref: '#/definitions/SelfLink'
-          - $ref: '#/definitions/PaginationLinks'
-      meta:
-        $ref: '#/definitions/Meta'
       data:
-        type: array
-        items:
-          $ref: '#/definitions/PetResource'
-  Error:
+        properties:
+          applications:
```

Comment:
I think `applications` should be a property of the `ApplicationResource` within its `attributes`

Reply:
Right, there were some mistakes. Will fix.